src/ir/context.rs

```diff
@@ -2430,17 +2437,27 @@ If you encounter an error missing from this list, please file an issue or a PR!"
         self.allowlisted = Some(allowlisted);
         self.codegen_items = Some(codegen_items);
 
+        let mut warnings = Vec::new();
+
         for item in self.options().allowlisted_functions.unmatched_items() {
-            warn!("unused option: --allowlist-function {}", item);
+            let warn = format!("unused option: --allowlist-function {}", item);
+            warn!("{}", warn);
```

Comment:
I am just notating here that this PR does not stop emitting warnings the usual way (something that was not initially clear to me when reading the PR description).